packages/util-endpoints/src/utils/callFunction.ts

```diff
@@ -5,5 +5,5 @@ import { evaluateExpression } from "./evaluateExpression";
 export const callFunction = ({ fn, argv }: FunctionObject, options: EvaluateOptions): FunctionReturn => {
   const argvArray = argv.map((arg) => (typeof arg === "boolean" ? arg : evaluateExpression(arg, "arg", options)));
   // @ts-ignore Element implicitly has an 'any' type
-  return lib[fn](...argvArray);
+  return fn.split(".").reduce((acc, key) => acc[key], lib)(...argvArray);
```

Comment:
minor naming issue: argvArray is "argument vector array", could instead be something like "evaluatedArguments"